feathr_project/feathr/sink.py

```diff
@@ -39,4 +39,44 @@ def to_feature_config(self) -> str:
             }
         """)
         msg = tm.render(source=self)
-        return msg
\ No newline at end of file
+        return msg
+
+
+class OfflineSink(Sink):
```

Comment:
Should we call it HDFSSink instead? Since we might also need to sink it to SQL or other offline stores (maybe in the future depending on user feedback)

Reply:
fixed.